src/contracts/PreAMMBatcher.sol

```diff
@@ -404,4 +404,31 @@ contract PreAMMBatcher {
         }
         return newOrders;
     }
+
+    function isSorted(Order[] memory orders, bool reverse)
```

Comment:
nit: `isSorted` and `reverse` is a bit ambiguous - to me `isSorted` doesn't indicate by what it is being sorder and `reverse` means that the orders will be checked in reverse order, but is not indicative of the sorting direction:

 ```suggestion
    enum Direction { Ascending, Descending }
    function isSortedByLimitPrice(Order[] memory orders, Direction direction)
```

Reply:
I am not sure that the contract will allow this, but I can try it out.

Reply:
https://solidity.readthedocs.io/en/v0.7.1/structure-of-a-contract.html#enum-types

Reply:
Awesome.